From 8515fe14dab7577d518bbb85484bd72324adbfe7 Mon Sep 17 00:00:00 2001
From: Stanley.Miao <stanley.miao@windriver.com>
Date: Thu, 28 Jul 2011 10:55:50 +0800
Subject: [PATCH] Add Graphics_SDK support for omap3630 platform

omap3630 uses SGX core revision 1.2.5 and it need a bigger
SYS_OMAP3430_SGX_REGS_SIZE.

Signed-off-by: Stanley.Miao <stanley.miao@windriver.com>
---
 drivers/gpu/pvr/omap3/sysconfig.h |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/drivers/gpu/pvr/omap3/sysconfig.h b/drivers/gpu/pvr/omap3/sysconfig.h
index c1f5e10..7495f2c 100644
--- a/drivers/gpu/pvr/omap3/sysconfig.h
+++ b/drivers/gpu/pvr/omap3/sysconfig.h
@@ -47,7 +47,11 @@
 
 #define SYS_OMAP3430_SGX_REGS_SYS_PHYS_BASE  0x50000000
 
+#if SGX_CORE_REV == 121
 #define SYS_OMAP3430_SGX_REGS_SIZE           0x4000
+#else
+#define SYS_OMAP3430_SGX_REGS_SIZE           0x10000
+#endif
 
 #define SYS_OMAP3430_SGX_IRQ				 21
 
-- 
1.7.0.4

